### PR TITLE
Sl 24

### PR DESCRIPTION
**Pull request by ZMehanovic**
Initial layout and and view of event details, there are still missing functionalities such as communication with web service but that will be implemented after adding fragments for event edit and create

### CODE REVIEW

File: app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -36,4 +46,20 @@
...
 
+    <style name="bold_text" parent="AppTheme">
+        <item name="android:textStyle">bold</item>
+    </style>
+
+    <style name="horizontal_line" parent="AppTheme">
+        <item name="android:layout_width">0dp</item>
+        <item name="android:layout_height">2dp</item>
+        <item name="android:layout_marginTop">8dp</item>
+        <item name="android:background">#888888</item>
```
Comment (vglozinic) [inline app/src/main/res/values/styles.xml:57]: Should be used as Android color or made a color into colors.xml

File: app/src/main/res/layout/fragment_event_details.xml [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+            <TextView
+                android:id="@+id/tv_title"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="16dp"
+                android:layout_marginTop="8dp"
+                android:layout_marginEnd="16dp"
+                android:gravity="start"
+                android:text="@{event.title}"
+                android:textColor="#000000"
```
Comment (vglozinic) [inline app/src/main/res/layout/fragment_event_details.xml:44]: Would be good to use android/black for colors

File: app/src/main/res/values/styles.xml [hunk 3]
```diff
@@ -1,14 +1,24 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="AppTheme" parent="Theme.AppCompat.Light">
```
Comment (gabilic) [inline app/src/main/res/values/styles.xml:4]: This shouldn't be changed, since we're gonna need the ActionBar later in application.

**Review by vglozinic — APPROVED** [review]
Some small changes are required, but generally it seems OK

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -1,13 +1,33 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (gabilic) [inline app/src/main/res/values/strings.xml:1]: String names should be adjusted to the naming convention.

File: app/src/main/res/values/styles.xml [hunk 5]
```diff
@@ -1,14 +1,24 @@
...
         <item name="android:textColor">@android:color/white</item>
         <item name="android:textSize">30sp</item>
     </style>
+    <style name="style_event_details_button" parent="style_button">
+        <item name="android:layout_width">0dp</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:layout_marginStart">16dp</item>
+        <item name="android:layout_marginTop">16dp</item>
+        <item name="android:layout_marginEnd">16dp</item>
+        <item name="android:textColor">@android:color/white</item>
```
Comment (gabilic) [inline app/src/main/res/values/styles.xml:18]: Items should be ordered alphabetically.

**Review by gabilic — APPROVED** [review]
Changes in SL-24 approved. Commented lines are gonna be reworked in improvement epic.

**Pull request merged**